snap/snapcraft.yaml

```diff
@@ -347,7 +296,7 @@ parts:
        "$SNAPCRAFT_PART_INSTALL/config/export-distro/res/configuration.toml"
 
       cat "./cmd/support-logging/res/configuration.toml" | \
-        sed -e s:./logs/edgex-support-logging.log:/var/snap/edgexfoundry/common/support-logging.log: \
+        sed -e s:./logs/support-logging.log:/var/snap/edgexfoundry/common/support-logging.log: \
```

Comment:
Note that I filed an issue for this logfile being different from the rest: https://github.com/edgexfoundry/edgex-go/issues/545